### ISSUE TITLE
"Select from Gallery" option on media file uploader does not use correct allowable file types

### ISSUE THREAD

**Issue by gwharton** [magento/magento2#23881]
### Preconditions (*)
1. Magento 2.4-develop vanilla

### Steps to reproduce (*)

1. Content -> Configuration
2. Global -> Edit
3. HTML Head -> Favicon icon
4. Select from Gallery
5. Choose Files
6. Select "<anyfile>.ico" from local filesystem

### Expected result (*)
1. icon file is successfully uploaded

### Actual result (*)
1. Error "We don't recognize or support this file extension type." is reported
![Peek 2020-08-31 13-22](https://user-images.githubusercontent.com/51681379/91710695-5b209700-eb8d-11ea-9ca1-9f434a435bf7.gif)

### Investigation
Allowed file types is stated "JPG, GIF, PNG, ICO, APNG" on design config webpage
Open Browser Dev tools and monitor network traffic when hitting "Select from Gallery" button.
Inspect inital ajax request's response back from server

```
allowedExtensions = 'jpg jpeg png gif';
acceptFileTypes: /(\.|\/)(gif|jpe?g|png)$/i,

```
May be fixed https://github.com/magento/magento2/pull/25007

**Comment by engcom-Bravo** [comment]
Not reproduced on 2.2-develop.

**Comment by magento-engcom-team** [comment]
:white_check_mark: Confirmed by @engcom-Bravo 
 Thank you for verifying the issue. Based on the provided information internal tickets `MC-18652` were created

**Issue Available**: @engcom-Bravo, _You will be automatically unassigned. Contributors/Maintainers can claim this issue to continue. To reclaim and continue work, reassign the ticket to yourself._

**Comment by rajeshwebkul** [comment]
@magento-engcom-team give me 2.3-develop instance

**Comment by magento-engcom-team** [comment]
Hi @sagarwebkul. Thank you for your request. I'm working on Magento 2.3-develop instance for you

**Comment by magento-engcom-team** [comment]
Hi @sagarwebkul, here is your [Magento instance](https://i-23881-2-3-develop.instances.magento-community.engineering).
Admin access: [https://i-23881-2-3-develop.instances.magento-community.engineering/admin](https://i-23881-2-3-develop.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`
Instance will be terminated in up to 3 hours.

**Comment by magento-engcom-team** [comment]
:white_check_mark: Confirmed by @engcom-Delta 
 Thank you for verifying the issue. Based on the provided information internal tickets `MC-37225` were created

**Issue Available**: @engcom-Delta, _You will be automatically unassigned. Contributors/Maintainers can claim this issue to continue. To reclaim and continue work, reassign the ticket to yourself._

**Comment by ihor-sviziev** [comment]
@sidolov @slavvka @naydav could you explain why this issue was closed without any explanations?

**Comment by sidolov** [comment]
@ihor-sviziev it was closed by mistake due to the issue with automation. Now it's resolved, Issue re-opened

**Comment by magento-engcom-team** [comment]
:white_check_mark: Confirmed by @engcom-Oscar 
 Thank you for verifying the issue. Based on the provided information internal tickets `MC-37690` were created

**Issue Available**: @engcom-Oscar, _You will be automatically unassigned. Contributors/Maintainers can claim this issue to continue. To reclaim and continue work, reassign the ticket to yourself._

**Comment by itsolon** [comment]
i  have same issue on php 7.4 und Magento 2.4.0 fresh install. FYI php7.4-fileinfo installed and active, and upload jpg png etc. working. only ico favicons NOT

**Comment by gwharton** [comment]
It is stale because nobody is working on a fix.

**Comment by ihor-sviziev** [comment]
let's keep it open

**Comment by magento-engcom-team** [comment]
Unfortunately, we are archiving this ticket now as it did not get much attention from both Magento Community and Core developers for an extended period. This is done in an effort to create a quality, community-driven backlog which will allow us to allocate the required attention more easily.

Please feel free to comment or reopen according to the [Issue reporting guidelines](https://github.com/magento/magento2/wiki/Issue-reporting-guidelines) 
the ticket if you are still facing this issue on the latest `2.x-develop` branch. Thank you for collaboration.

**Comment by magento-engcom-team** [comment]
Hi @gwharton. 

Thank you for your report and collaboration!

The related internal Jira ticket `MC-37225` was closed as `Fixed`. 

The fix will be available with the upcoming `2.4.3` release.

**Comment by magento-engcom-team** [comment]
Hi @gwharton. 

Thank you for your report and collaboration!

The related internal Jira ticket `MC-23892` was closed as `Fixed`. 

The fix will be available with the upcoming `2.4.3` release.

**Comment by magento-engcom-team** [comment]
Hi @gwharton. 

Thank you for your report and collaboration!

The related internal Jira ticket `MC-18652` was closed as `Fixed`. 

The fix will be available with the upcoming `2.4.3` release.
